lib/const.rb

```diff
@@ -46,19 +46,21 @@
 # downloaded, e.g. in a docker container.
 ARCH_LIB = ARCH.eql?('x86_64') && Dir.exist?('/lib64') ? 'lib64' : 'lib'
 
+libc_path = `whereis libc.so.6`[11, 1000]
+
 # Glibc version can be found from the output of libc.so.6
-LIBC_VERSION = `/#{ARCH_LIB}/libc.so.6`[/Gentoo ([^-]+)/, 1]
+LIBC_VERSION = `#{libc_path}`[/Gentoo ([^-]+)/, 1]
 
 if ENV['CREW_PREFIX'].to_s.empty? || (ENV['CREW_PREFIX'] == '/usr/local')
-  CREW_BUILD_FROM_SOURCE = ENV.fetch('CREW_BUILD_FROM_SOURCE', nil)
   CREW_PREFIX = '/usr/local'
   HOME = Dir.home
 else
-  CREW_BUILD_FROM_SOURCE = 1
```

Comment:
Don't we always want to build from source if `CREW_PREFIX != '/usr/local'`?

Reply:
Why would that be the case?

Reply:
All current binaries are built with `--prefix=/usr/local` - so if you change CREW_PREFIX, none of the binaries will work...